### ISSUE TITLE
Collect WG chairs' bio info and add to Leadership page 

### ISSUE THREAD

**Issue by nowviskie** [clirdlf/ndsa.org#46]
Just getting this over here as a ticket. Might want to change title of the page as well, and indicate which members are elected to the CC (with term information, which you may need to solicit). http://ndsa.org/coordinating-committee/

**Comment by ohbendorf** [comment]
Thanks- I will collect that information and add it to Leadership page, with reformatting as needed. Just to confirm, you are the sole interim Innovation chair, correct? I know that Jane has been continuing to be involved, but from what I understand her term ended in January. Finally, it is my understanding that Outreach is on hiatus and thus that I should not collect this information from Carol Minton-Morris- but if that is incorrect, please just let me know.

**Comment by nowviskie** [comment]
Correct on both counts! Thank you, Oliver!

**Comment by ohbendorf** [comment]
Bios and headshots are added- I'm waiting until Wayne is back to address a URL change. No term limits yet, but can gather. In my opinion, the fadeins are a bit more of a challenge now that there are a handful more people listed on the page... may be worth considering... fading them out...

**Comment by waynegraham** [comment]
You can do a replace for

```
 <div class="col-sm-10 animated fadeInLeft wow" data-wow-delay=".3s">
```

to

```
 <div class="col-sm-10">
```

This is controlled through the `animated` and `wow` classes (the `data-wow-delay` overrides the default transitions.

**Comment by waynegraham** [comment]
This is closed as of 4cb74e5
